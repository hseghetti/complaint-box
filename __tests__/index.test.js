import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../pages/index';

describe('Home Component', () => {
  it('Should render Home', () => {
    const component = renderer.create(
      <Home />
    );
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});