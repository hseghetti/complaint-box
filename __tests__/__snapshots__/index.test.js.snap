// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Should render Home 1`] = `
Array [
  <button
    className="chakra-button css-10iqt4h"
    onClick={[Function]}
    type="button"
  >
    <svg
      className="chakra-icon css-onkibi"
      focusable={false}
      viewBox="0 0 24 24"
    >
      <g
        fill="none"
        stroke="currentColor"
        strokeLinecap="round"
        strokeLinejoin="round"
        strokeWidth="2"
      >
        <circle
          cx="12"
          cy="12"
          r="5"
        />
        <path
          d="M12 1v2"
        />
        <path
          d="M12 21v2"
        />
        <path
          d="M4.22 4.22l1.42 1.42"
        />
        <path
          d="M18.36 18.36l1.42 1.42"
        />
        <path
          d="M1 12h2"
        />
        <path
          d="M21 12h2"
        />
        <path
          d="M4.22 19.78l1.42-1.42"
        />
        <path
          d="M18.36 5.64l1.42-1.42"
        />
      </g>
    </svg>
  </button>,
  <div
    className="container"
  >
    <main
      className="main"
    >
      <div
        className="css-txajem"
        style={
          Object {
            "left": 0,
            "maxWidth": "100vw",
            "position": "static",
            "right": 0,
            "top": 0,
            "transform": "translateY(-100%)",
            "transition": "all 1000ms cubic-bezier(0, 0, 0.2, 1)",
            "willChange": "transform",
          }
        }
      >
        <form
          action="#"
          onReset={[Function]}
          onSubmit={[Function]}
        >
          <div
            className="chakra-form-control css-1idxhhb"
            role="group"
          >
            <label
              className="chakra-form__label css-v80092"
              htmlFor="comment"
              id="field-1-label"
            >
              Comment 
              <svg
                className="chakra-icon css-onkibi"
                focusable={false}
                viewBox="0 0 24 24"
              >
                <g
                  fill="none"
                  stroke="currentColor"
                  strokeLinecap="round"
                  strokeWidth="2"
                >
                  <path
                    d="M11 4H4a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14a2 2 0 0 0 2-2v-7"
                  />
                  <path
                    d="M18.5 2.5a2.121 2.121 0 0 1 3 3L12 15l-4 1 1-4 9.5-9.5z"
                  />
                </g>
              </svg>
            </label>
            <textarea
              className="chakra-textarea css-12oetzt"
              disabled={false}
              id="comment"
              name="comment"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              placeholder="Please complete your comment"
              readOnly={false}
              required={false}
              value=""
            />
          </div>
          <button
            className="chakra-button css-1fwt0py"
            disabled={false}
            type="submit"
          >
            Send
            <svg
              className="chakra-icon css-1m88xoa"
              focusable={false}
              viewBox="0 0 24 24"
            >
              <g
                fill="currentColor"
              >
                <path
                  d="M11.114,14.556a1.252,1.252,0,0,0,1.768,0L22.568,4.87a.5.5,0,0,0-.281-.849A1.966,1.966,0,0,0,22,4H2a1.966,1.966,0,0,0-.289.021.5.5,0,0,0-.281.849Z"
                />
                <path
                  d="M23.888,5.832a.182.182,0,0,0-.2.039l-6.2,6.2a.251.251,0,0,0,0,.354l5.043,5.043a.75.75,0,1,1-1.06,1.061l-5.043-5.043a.25.25,0,0,0-.354,0l-2.129,2.129a2.75,2.75,0,0,1-3.888,0L7.926,13.488a.251.251,0,0,0-.354,0L2.529,18.531a.75.75,0,0,1-1.06-1.061l5.043-5.043a.251.251,0,0,0,0-.354l-6.2-6.2a.18.18,0,0,0-.2-.039A.182.182,0,0,0,0,6V18a2,2,0,0,0,2,2H22a2,2,0,0,0,2-2V6A.181.181,0,0,0,23.888,5.832Z"
                />
              </g>
            </svg>
          </button>
        </form>
      </div>
    </main>
    <footer
      className="footer"
    >
      <div
        className="css-77qi67"
      >
        <div
          className="css-16zircp"
        />
      </div>
    </footer>
  </div>,
]
`;
